actor-typed/src/main/scala/org/apache/pekko/actor/typed/internal/ActorContextImpl.scala

```diff
@@ -284,7 +284,7 @@ import scala.util.Success
   def pipeToSelf[Value](
       future: CompletionStage[Value],
       applyToResult: pekko.japi.function.Function2[Value, Throwable, T]): Unit = {
-    future.whenComplete { (value, ex) =>
+    future.handle[Unit] { (value, ex) =>
```

Comment:
Is it necessary to make changes? If so, why hasn't PipeToSupport made improvements in this regard?

Reply:
Yes, `handle` is faster than `whenComplete`, especially in the failed case.